src/neo/Consensus/ConsensusContext.cs

```diff
@@ -24,6 +26,8 @@ internal class ConsensusContext : IDisposable, ISerializable
         private const byte ConsensusStatePrefix = 0xf4;
 
         public Block Block;
+        public UInt256 ProposalStateRoot { get; set; }
```

Comment:
There should not be a `ProposalStateRoot`,
we have `PreviousBlockStateRoot` and `CurrentStateRoot. The latter is relayed as a message.